examples/random-colors.php

```diff
@@ -23,10 +19,10 @@
 }
 
 // process control codes from STDIN
-$stdin = new ReadableResourceStream(STDIN);
-$parser = new ControlCodeParser($stdin);
+$stdin = new eact\Stream\ReadableResourceStream(STDIN);
```

Comment:
```suggestion
$stdin = new React\Stream\ReadableResourceStream(STDIN);
```

Reply:
Oops, thank you for noticing this. Should be fixed now. 🎉